.github/workflows/publish-package.yml

```diff
@@ -0,0 +1,68 @@
+name: publish distributions
+on:
+  push:
+    branches:
+    - master
+    tags:
+    - v*
+  pull_request:
+    branches:
+    - master
```

Comment:
@dguest Note that as these are set to `master` this workflow won't run until Issue #28 is resolved.